include/CppUTest/SimpleString.h

```diff
@@ -93,6 +93,8 @@ class SimpleString
     static void setStringAllocator(TestMemoryAllocator* allocator);
 
     static int AtoI(const char*str);
+    static unsigned AtoU(const char*str, bool& wasAbleToParseADigit);
```

Comment:
Would you mind returning 0 on failure? I believe that works and then the output boolean is not needed. I know it works like this, just wouldn't really want to have output boolean parameters :)

Reply:
If 0 value is used to signal error, `shuffleDisabledCase3` stops working (i.e., `./testExecutable -s 0` does not run any tests). --> I would say the output boolean is mandatory, and it needs to be kept.

Reply:
But... 0 is an invalid seed in your code anyways... so -s 0 would be an invalid command already now.

Reply:
I changed the behavior of the command line parser to treat `-s0` (and `-s 0`) as invalid parameters, and was able to remove the output boolean.